clustered/clustered-dist/build.gradle

```diff
@@ -119,6 +114,7 @@ distributions {
       }
       into ("server/plugins/lib") {
         from (configurations.serverLibs - configurations.serverApis)
+        duplicatesStrategy DuplicatesStrategy.EXCLUDE
```

Comment:
You'd be better off playing the trick that I use in the enterprise kit and applying this duplicatesStrategy at a higher level, but only for jar files.

Reply:
used :
```
      filesMatching('**/*.jar') {
        // We can safely exclude JAR duplicates as our dependency strategy is fail on conflict
        duplicatesStrategy DuplicatesStrategy.EXCLUDE
      }
```